scalalib/src/ScalaModule.scala

```diff
@@ -104,6 +104,14 @@ trait ScalaModule extends JavaModule { outer =>
    */
   protected def mandatoryScalacOptions = T { Seq.empty[String] }
 
+  /**
+   * Scalac options to active the compiler plugins.
+   */
+  protected def enablePluginScalacOption: Target[Seq[String]] = T {
```

Comment:
What do you think of making this `private` until we're sure this is the way to go? Otherwise we will need to maintain this for binary compatibility until `0.11`.
Also I don't get the naming. Should it be plural like `enablePluginScalacOptions` ?
What about something like `compilerPluginsScalacOptions` ?

Reply:
You're early to the party. I'm still editing, but also juggling with family time and waiting for CI. The plural form is already fixed. The private makes sense. The enable because this is not "all" plugin options but only those we infer from the ivy deps.

Reply:
> You're early to the party.

Definitely! It felt almost like pair programming :-)
I'll leave you until it is ready for review :)

Reply:
haha. We can try pair programming some time tough.

Reply:
I thought about the name that you suggested but discarded it because of my argument above.